requirements.txt

```diff
@@ -0,0 +1,7 @@
+click==6.7
```

Comment:
Can you briefly comment on what each package is and why you included it?

Reply:
They are all the things that you get from `pip install flask`.
It's good to just `pip freeze > requirements.txt` so that everything is pinned.

Reply:
So for everyone else: we want to use [Flask](http://flask.pocoo.org/) as our web framework. That package itself depends on other ones, which you can see in this file. We are installing the latest versions, which we pin here using `==` so that everyone runs the app with the exact same code.